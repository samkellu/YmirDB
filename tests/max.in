max a
set a 1 2 3 4 5 6 22 34
set b 123 334 a 44
set c b a 12 55 7
max c
bye
