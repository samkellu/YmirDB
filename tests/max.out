> No such entry

> ok

> ok

> no such key

> No such entry

> bye
